apps/glances/config.json

```
@@ -0,0 +1,19 @@
+{
+  "$schema": "../schema.json",
+  "name": "Glances",
+  "port": 8420,
+  "available": true,
+  "exposable": true,
```

```suggestion
  "exposable": false,
```
There should be no reason to expose this with a Public Domain IMO. Can confirm with @meienberger

---

Sounds good, no disagreement.